src/theme.c

```diff
@@ -362,47 +389,47 @@ load_buttons(struct theme *theme)
 		zdrop(b->inactive.buffer);
 
 		/* PNG */
-		snprintf(filename, sizeof(filename), "%s-active.png", b->name);
-		button_png_load(filename, b->active.buffer);
-		snprintf(filename, sizeof(filename), "%s-inactive.png", b->name);
-		button_png_load(filename, b->inactive.buffer);
+		get_button_filename(b->name, "-active.png", filename, sizeof(filename));
+		img_png_load(filename, b->active.buffer);
+		get_button_filename(b->name, "-inactive.png", filename, sizeof(filename));
```

Comment:
Bug: `get_button_filename()` doesn't touch `filename` if the button file is not found, so old `filename` is passed to `img_{svg,png,xbm}_load()` again and again. I'll fix it.